app/dashboard/views.py

```diff
@@ -1006,6 +1006,34 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
     return TemplateResponse(request, 'bounty/details.html', params)
 
 
+def get_notify_funder_modal(request, stdbounties_id):
+    stdbounties_id = clean_str(stdbounties_id)
+    bounty = Bounty.objects.filter(standard_bounties_id=stdbounties_id).first()
+
+    context = {
+        'bounty': bounty,
+        'active': 'get_notify_funder_modal',
+        'title': _('Send Payout Reminder')
+    }
+    return TemplateResponse(request, 'notifyfunder.html', context)
+
+
+@csrf_exempt
+def funder_payout_reminder(request, bounty_id):
+    try:
+        bounty = Bounty.objects.get(pk=bounty_id)
```

Comment:
> 

> If we use `network` and `standard_bounties_id`, we can use `Bounty.objects.current().filter(network=network, standard_bounties_id=stdbounties_id)`.
> 
> `.current()` makes sure that you get the latest revision of that bounty object

Should this line also be: 
```
    bounty = Bounty.objects.current().filter(network=network, standard_bounties_id=stdbounties_id).first()
```?

Reply:
@oogetyboogety 
Sure, if you change the endpoint to use `network` and `standard_bounties_id`, that should be updated too 👍